.travis.yml

```diff
@@ -1,53 +1,58 @@
 language: php
 sudo: false
+dist: trusty
```

Comment:
This is implied anyway isn't it now? I thought Travis had already migrated default to `trusty`?